src/lib.rs

```diff
@@ -321,6 +400,27 @@ mod tests {
     use crate::curv::arithmetic::traits::Samplable;
     use std::str;
 
+    #[test]
+    fn check_compressed_form(){
```

Comment:
perhaps show that compressed form is indeed 'compressed' compared to decompressed form

Reply:
👍

Reply:
Added the following assertion: 
`assert!(abd_comp.b_compressed.bit_length()` <= 10+b.bit_length() /2);`
The "+10" bits to this safety as it's not always half the space (in most cases b is around 1000 bits and compressed_b is around 500)